---
layout: about
title: about
permalink: /
description:

profile:
  align: right
  image: hesselberth-lab-hex-logo.png

news: true # includes a list of news items
selected_papers: false # includes a list of papers marked as "selected={true}"
social: true  # includes social icons at the bottom of the page
---

**[Department of Biochemistry and Molecular
Genetics](https://medschool.cuanschutz.edu/biochemistry) and [RNA Bioscience
Initiative](https://medschool.cuanschutz.edu/rbi), [University of Colorado
School of Medicine](https://www.cuanschutz.edu/)**

The broad goals of our laboratory are to:

- Understand how RNA damage and repair are integrated with stress responses
(funded by an [NIGMS
MIRA](https://reporter.nih.gov/search/FWmPa7DGuk6CmPb8Ocb5XQ/project-details/10193187))
- Develop new molecular methods that enhance single-cell and spatial gene
expression experiments (funded by an [NIH Transformative
R01](https://commonfund.nih.gov/tra/recipients))
- Develop and apply RNA therapeutics that modulate gene expression (funded
by the [RNA Bioscience Initiative](https://medschool.cuanschutz.edu/rbi))

We are a group of RNA biologists, technology developers, and data analysts on a
mission to discover fundamental principles of RNA regulation and apply them in
creative ways to improve human health.

A major effort in the lab focuses on the biology of RNA damage and repair using a
combination of method development, bioinformatics, genetics, biochemistry, and
cell biology. Over the past ten years we have developed [new sequencing
approaches](https://pubmed.ncbi.nlm.nih.gov/26001965/) to identify damaged RNAs
and [new model systems](https://pubmed.ncbi.nlm.nih.gov/29212664/) with which to
understand the causes and consequences of RNA damage. Our studies have led to
[new concepts](https://elifesciences.org/articles/42262) in post-transcriptional
regulation that have broad implications for understanding the role of RNA damage
in biology.

A second effort focuses on the development of novel technologies for
analysis of additional phenotypes in single-cell and spatial gene
expression approaches. Our recent advances include the [measurement of
enzyme activity and inhibition](https://pubmed.ncbi.nlm.nih.gov/32286626/)
in single-cell assays, [tracking of ligands and associated gene expression
responses](https://pubmed.ncbi.nlm.nih.gov/33843587/) through animal
tissue, and approaches for quantifying metabolite levels.

The newest effort in the lab is a collaboration with clinicians at
Colorado Children's Hospital to develop RNA therapies for treatment of
pediatric neurological diseases. We are developing a pipeline using
patient-derived cells to identify therapeutic antisense oligonucleotides
that correct the negative impacts of mutations that alter pre-mRNA
splicing.

Finally, we host the [Informatics
Fellows](https://github.com/rnabioco/)
of the RNA Bioscience Initiative, who develop and apply bioinformatic approaches
with investigators across our campus to enable widespread deployment of
new RNA analysis technologies.

